tests/README.md

```diff
@@ -121,9 +121,34 @@ This allows run by case filter to simulate CI. Anybody can customize the case la
   * Export the token
     * `export RHCS_TOKEN=<rhcs token>` #If it had been run, then you can skip
   * Export the label filter to locate the preparation case
-    * `export LabelFilter="destroy"` #This filter only choose all critical and high priority cases with day1-post and day2
+    * `export LabelFilter="destroy"` #This filter only choose the destroy case for resources destroy
   * Run make command
     * `make e2e_test`
+### To run on local simulate CI without make file
+This allows run by case filter to simulate CI. Anybody can customize the case label filter to filter out the cases want to run
```

Comment:
I didn't understand it. 
@EricPonvelle can you rephrase it or help @xueli181114 with it?

Reply:
Maybe something like that: 
```suggestion
### Running a local CI simulation
This feature allows for running tests through a case filter to simulate CI. Anyone can customize the case label filter to select the specific cases that would be run. 
```